web/spec/supabase.yml

```diff
@@ -853,6 +853,8 @@ pages:
   removeSubscription():
     title: 'removeSubscription()'
     $ref: '@supabase/supabase-js."SupabaseClient".SupabaseClient.removeSubscription'
+    notes: |
+      - Removing subscriptions will keep your Realtime Elixir server performant. Failing to do so will increase message latency and eventually cause message delivery failures as more of your users are simultaneously subscribed.
```

Comment:
Great call out @w3b6x9 

I think we might need to keep the language a bit less technical - many people won't know what a Realtime Elixir server is if they jump directly to this part of the docs. Perhaps something like (feel free to re-word)

"Removing subscriptions is a great way to maintain the performance of your database {*project??*}. Supabase will automatically handle cleanup after 30(?) seconds, but unused connections may cause degradation as more users are simultaneously subscribed."